### PR TITLE
[Genius] fix: lyrics

### LINKED ISSUE #3185: [Bug | Presence] Genius
**Describe the Bug**

No details are displayed with the Genius presence.

**Expected Behavior**

Normally, details should be displayed (e.g.; what lyrics I'm looking at etc...).

**Screenshots**

https://media.discordapp.net/attachments/815922844990177331/817722969420988436/unknown.png

**Environment**

- OS: Windows 10
- Browser: Edge **(<- I think it's related)**
- Version:  89.0.774.45 (Official build) (64-bit)

**Additional context**

I tested several things like : change the language of the presence, change the language of my computer, change DNS but nothing happens.

---
PreMiD ; v2.2.0
Extension : v2.2.3 (from Microsoft Edge Addons)

### PR DESCRIPTION
**Pull request by mlunax**
Resolves #3185

### CODE REVIEW

**Review by ririxidev — DISMISSED** [review]

File: websites/G/Genius/presence.ts [hunk 1]
```diff
@@ -87,12 +87,12 @@ presence.on("UpdateData", async () => {
           url: document.URL
         }
       ];
-  } else if (document.querySelector(".song_body-lyrics") !== null) {
+  } else if (document.querySelector("div[class*='SongPageGrid'") !== null) {
```
Comment (Bas950) [inline websites/G/Genius/presence.ts:90]:
```suggestion
  } else if (document.querySelector("div[class*='SongPageGrid'") !== null || document.querySelector(".song_body-lyrics") !== null) {
```

File: websites/G/Genius/presence.ts [hunk 2]
```diff
@@ -87,12 +87,12 @@ presence.on("UpdateData", async () => {
...
     const song = document
-        .querySelector("h1.header_with_cover_art-primary_info-title")
+        .querySelector("h1[class*='SongHeader__Title-sc'")
         .textContent.trim(),
```
Comment (Bas950) [inline websites/G/Genius/presence.ts:93]:
```suggestion
        .querySelector("h1[class*='SongHeader__Title-sc'")
        ?.textContent.trim() || document
        .querySelector("h1.header_with_cover_art-primary_info-title")
        ?.textContent.trim(),
```

File: websites/G/Genius/presence.ts [hunk 3]
```diff
@@ -87,12 +87,12 @@ presence.on("UpdateData", async () => {
...
       artist = document
-        .querySelector("a.header_with_cover_art-primary_info-primary_artist")
+        .querySelector("a[class*='SongHeader__Artist'")
         .textContent.trim();
```
Comment (Bas950) [inline websites/G/Genius/presence.ts:96]:
```suggestion
        .querySelector("a[class*='SongHeader__Artist'")
        ?.textContent.trim() || document
        .querySelector("a.header_with_cover_art-primary_info-primary_artist")
        ?.textContent.trim();
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
For me the old ones worked, some regions have different elements...

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**